apps/client/src/app/userId/[id]/page.tsx

```diff
@@ -0,0 +1,17 @@
+import QRCode from "react-qr-code";
+
+export default function Index({ params }: {
+    params: {id: string}
```

Comment:
Currently we do not need to pass user id as params, so can we just create a variable `userId` inside component instead

Reply:
Done.

Reply:
Awesome!